app/assets/stylesheets/govuk_publishing_components/components/_chat-entry.scss

```diff
@@ -3,6 +3,8 @@
 .gem-c-chat-entry {
   display: flex;
   align-items: flex-start;
+  background-color: #e8f1f8; // there is no Design System color palette option or Sass variable defined for #d2e1f1
```

Comment:
I think you've missed updating the comment when this colour changed.

Reply:
Thanks, yep good spot. I've updated.